surveys/serializers.py

```diff
@@ -0,0 +1,68 @@
+from rest_framework import serializers
+
+from . import models
+
+
+class SurveyFieldSerializer(serializers.ModelSerializer):
+    """For a single field.  Nested into SurveyFieldsetSerializer."""
+    class Meta:
+        model = models.SurveyField
+        fields = ['name', 'help_text', 'field_type', 'answers', 'required']
+
+
+class SurveyFieldsetSerializer(serializers.ModelSerializer):
+    """A complete fieldset.  Nested into SurveySerializer but also works alone."""
+    fields = SurveyFieldSerializer(many=True)
+
+    class Meta:
+        model = models.SurveyFieldset
+        fields = ['name', 'description', 'fields']
+
+
+class SurveySerializer(serializers.ModelSerializer):
+    """Serializer for a full survey."""
+    fieldsets = SurveyFieldsetSerializer(many=True)
+
+    class Meta:
+        model = models.Survey
+        fields = ['name', 'description', 'fieldsets']
+
+
+class UserResponseSerializer(serializers.ModelSerializer):
+    """
+    Serializer for a user's response to one fieldset, but without a survey required.
+
+    Intended only for use as nested into SurveyResponseSerializer.
+    """
+    class Meta:
+        model = models.UserResponse
+        fields = ['fieldset', 'user_id', 'answers']
+
+
+class SurveyResponseSerializer(serializers.Serializer):
+    """Serializer for a user's answers to a whole survey."""
+    user_responses = UserResponseSerializer(many=True)
+    survey = serializers.PrimaryKeyRelatedField(queryset=models.Survey.objects.all())
```

Comment:
If we want to remove survey, we can use instead a `ModelSerializer` as we have the same information in the url, eg. `http://localhost:8000/forms/1`